import-automation/progress-dashboard-rest/test/test_base_database.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 Google LLC
```

Comment:
Typically, the test file for foo.py is named foo_test.py.  Also, typically they are in the same directory.  Thus, one can just list a directory's contents to see which files have tests and which do not.  If data files for testing are needed, we might put them into a "tests/" subdirectory.

Reply:
unittest assumes test files have the prefix test_